src/routes/AppRoutes.js

```diff
@@ -7,8 +7,8 @@ import Loadable from "ui-component/Loadable";
 // overview routing
 const Dashboard = Loadable(lazy(() => import("views/pages/dashboard")));
 const Activity = Loadable(lazy(() => import("views/pages/activity")));
-const Groups = Loadable(lazy(() => import("views/pages/groups")));
-const Group = Loadable(lazy(() => import("views/pages/group")));
+const Groups = Loadable(lazy(() => import("views/pages/groups/groups")));
```

Comment:
change the groups.js file to index.js to clean this up (see example of dashboard)